### PR TITLE
Geschichte

### PR DESCRIPTION
**Pull request by falkkreimeier**

### CODE REVIEW

File: neuerordner/Inhaltsverzeichnis.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Inhalt 
+
+-Kapitel 1
```
Comment (mbosselmann) [inline neuerordner/Inhaltsverzeichnis.md]: maybe subheadings?
```suggestion
## -Kapitel 1
```

File: neuerordner/Inhaltsverzeichnis.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    - Ninja Cat lernt den Roundhousekick
+
+-Kapitel 2
```
Comment (mbosselmann) [inline neuerordner/Inhaltsverzeichnis.md]:
```suggestion
## -Kapitel 2
```

File: neuerordner/Inhaltsverzeichnis.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    - Trump geht Ninja Cat mächtig auf den Sack
+
+-Kapitel 3
```
Comment (mbosselmann) [inline neuerordner/Inhaltsverzeichnis.md]:
```suggestion
## -Kapitel 3 
```

File: neuerordner/Inhaltsverzeichnis.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+-Kapitel 1
+    - Ninja Cat lernt den Roundhousekick
+
+-Kapitel 2
+    - Trump geht Ninja Cat mächtig auf den Sack
+
+-Kapitel 3 
+    - Ninja Cat roundhousekicked Trump aus dem weißen Haus
+    
+-Kapitel 4
```
Comment (mbosselmann) [inline neuerordner/Inhaltsverzeichnis.md:12]:
```suggestion
## -Kapitel 4
```

File: neuerordner2/Neue Geschichte  [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# Mörchenhaft
```
Comment (mbosselmann) [inline neuerordner2/Neue Geschichte ]: wrong spelling
```suggestion
# Märchenhaft
```

**Review by mbosselmann — CHANGES_REQUESTED** [review]
Wonderful content. I just have some minor changes, you can consider, if you like. :-)

**Review by lars-radtke — APPROVED** [review]
I think the idea of Ninja Cat is pretty dope!

**Comment by StefanBehring** [comment]
This is disturbing! Can't wait to read more!

**Pull request merged**